celavi/component.py

```diff
@@ -142,12 +140,51 @@ def bol_process(self, env):
 
         # component waits to be manufactured
         yield env.timeout(begin_timestep)
+
+        # Identify manufacturing facility based on distance and, for secondary manuf 
+        # facilities, whether the facility has sufficient inventory to manufacture the
+        # component
+        
+        # Locate the closest (by cost) manufacturing facilities
+        _manuf_dict = self.context.cost_graph.find_upstream_neighbor(
+            node_id = self.in_use_facility,
+            crit = 'cost', # @NOTE Replace with 'dist' to look at location only, not processing costs
```

Comment:
SO shouldn't it be dist then? I see the comment a coupld lines below mentions distance as well, not costs.

Reply:
I use "distance" in a couple places to refer to whichever criteria is summed along the paths between nodes. We can use literal distance (km) or costs as that criteria, although cost is the default. For this logic, I think costs make more sense, although if we were only dealing with virgin manufacturing then distance would give equivalent results.